### PR TITLE
handle archive feature toggling with css

### PR DESCRIPTION
**Pull request by jdrouet**
Signed-off-by: Jérémie Drouet <jeremie.drouet@gmail.com>

### Proposed changes

Toggle archive features only with css to fix race condition around loading `/js/archives.json`

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -62,7 +62,6 @@
   <script>
   // Default to assuming this is an archive and hiding some stuff
   // See js/archive.js and js/docs.js for logic relating to this
-  var isArchive = true;
   var dockerVersion = 'v{{ site.docker_ce_version }}';
```
Comment (thaJeztah) [inline _includes/head.html:65]: not for this PR, but this one kept confusing me (we're setting it both here, and in the `_body.tml`

File: js/archive.js [hunk 2]
```diff
@@ -6,28 +6,45 @@ layout: null
...
-                               '<span style="z-index: 1001" class="dropdown">';
-    var listStart = '<ul class="dropdown-menu" role="menu" aria-labelledby="archive-menu">';
-    var listEnd = '</ul>';
-    var outerDivEnd = '</span></div></div></div>';
+  $.getJSON("/js/archives.json", function (result) {
+    var outerDivStart =
+      '<div id="archive-butterbar"><div class="container"><div style="text-align: center"><span id="archive-list">This is <b><a href="https://docs.docker.com/docsarchive/" style="color: #254356; text-decoration: underline !important">archived documentation</a></b> for Docker&nbsp;' +
+      dockerVersion +
+      '. Go to the <a style="color: #254356; text-decoration: underline !important" href="https://docs.docker.com/">latest docs</a> or a different version:&nbsp;&nbsp;</span>' +
```
Comment (thaJeztah) [inline js/archive.js:13]: So, if we're not gonna be publishing the current (19.03) version of the docs as an archive, do we need this script at all? Basically this script would only be actively running on archived versions

Comment (usha-mandya) [reply js/archive.js:13]: Yup, that's correct. As per the earlier discussions, the current version will not be archived and there will be only one version of the docs on docs.docker.com

**Comment by usha-mandya** [comment]
@thaJeztah As we have a plan to create a new PR to remove the archives, should we merge this one? The fixes in this PR impact https://github.com/docker/docker.github.io/pull/10967#issuecomment-641093607

**Pull request merged**